### ISSUE TITLE
Multiple "Use as tempo"

### ISSUE THREAD

**Issue by manucontrovento** [openAVproductions/openAV-Luppp#288]
Hi everyone!

I'll start posting here my suggestions.
It would be nice to have the Use as tempo feature on more than one clip. This would be useful to load more than one "song" in a session. I could load 3 or 4 songs, using a "row" for each song, with different BPM, since the tempo would be given by the loop I select. Or you could manage a song which changes its tempo.

greetings!
manu

**Comment by georgkrause** [comment]
So you basically want to set different speed for each scene?

**Comment by manucontrovento** [comment]
Yes; the more flexible behaviour would be, LuPPP to change BPM according to what is triggered (clip or whole scene). Maybe, as optional, so as an option in the application preferences.
But even just a scene-triggering bpm tempo change would be nice.
cheers

**Comment by georgkrause** [comment]
while this is in general possible, this needs a lot of design. like: what happens if i have a clip running in scene 1 with 120bpm and the user launches a clip in scene 2 with 135 bpm. When will it start? what happens with the clips in scene 1?

**Comment by manucontrovento** [comment]
Ok, I understand what you mean. Therefore, the best thing should be, assign to a scene a bpm property, and triggering the scene, LuPPP tempo will switch to the scene tempo.

**Comment by georgkrause** [comment]
This basically means its not possible anymore to play two clips from different scenes, which would take a lot of flexibility. I need to think about how to solve this...

**Comment by manucontrovento** [comment]
Ok I understand. But I think that the current behaviour of LuPPP could remain the same. Maybe it could be added a "bpm" "optional" property to the scene, so if you want the scene to change the bpm of the session you have the chance. Otherwise you just trigger the scene as it happens now.
